vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -361,4 +428,20 @@ private void updateContentOrder(String slideId, Integer contentOrder) {
             contentBlockRepository.saveAll(contentBlockList);
         }
     }
+    
+    @Override
+    public HashMap<String, String> findSelectedSpaceForSpaceBlock(List<IContentBlock> slideContents){
+        HashMap<String, String> selectedSpaceForSpaceBlock = new HashMap<String, String>();
+        for(IContentBlock slide : slideContents) {
+            if(slide.getClass().getSimpleName().equals("SpaceBlock")) {
+                Optional<SpaceBlock> optionalSpaceBlock = spaceBlockRepo.findById(slide.getId());
+                if(optionalSpaceBlock.isPresent()) {
+                    selectedSpaceForSpaceBlock.put(slide.getId(),optionalSpaceBlock.get().getSpace().getName());
+                }
+            }
+            
+        }
+        return selectedSpaceForSpaceBlock;
```

Comment:
what is this method for? why does it retrieve space blocks again? (also if you. need to check the type of an object you should use `instanceof`).

Reply:
Removed this method as it was not needed